houndigrade/cli.py

```diff
@@ -0,0 +1,215 @@
+"""Command line script for inspecting attached volumes."""
+import glob
+import json
+import os
+import subprocess
+from contextlib import contextmanager
+from gettext import gettext as _
+
+import click
+from kombu import Connection, Exchange, Queue
+
+INSPECT_PATH = '/mnt/inspect'
+
+
+@click.command()
+@click.option('--cloud',
+              '-c',
+              default='aws',
+              help='Cloud in which we are performing the inspection.',
+              type=click.Choice(['aws', 'gcp', 'azure']))
+@click.option('--target',
+              '-t',
+              multiple=True,
+              type=(str, click.Path(exists=True)),
+              required=True,
+              help=_('Inspection target, the cloud specific image identifier'
+                     ' and path to the attached drive on the machine. e.g.'
+                     ' -t ami-12312839312 /dev/sda'))
+@click.option('--debug',
+              is_flag=True,
+              help=_('Print debug output.'))
+def main(cloud, target, debug):
+    """
+    Mounts provided volumes and inspects them.
+
+    The script takes provided target information and checks if there are any
+    partitions on that device, then it loops through, mounting every partition
+    and inspecting it. As it loops through it builds a dictionary of results
+    that gets placed on a queue once the processing is done.
+
+    """
+    click.echo(_('Provided cloud: {}'.format(cloud)))
+    click.echo(_('Provided drive(s) to inspect: {}'.format(target)))
+
+    results = {
+        'cloud': cloud,
+        'inspection_targets': target,
+        'facts': {},
+    }
+
+    for image_id, drive in target:
+        click.echo('Checking drive {}'.format(drive))
+        results['facts'][drive] = {'image_id': image_id}
+        for partition in get_partitions(drive):
+            click.echo('Checking partition {}'.format(partition))
+
+            results['facts'][drive][partition] = []
+            try:
+                with mount(partition, INSPECT_PATH):
+                    check_release_files(
+                        partition,
+                        results['facts'][drive][partition]
+                    )
+
+            except subprocess.CalledProcessError as e:
+                click.echo(
+                    _('Mount of {} failed '
```

Comment:
I _think_ this split string can fit on the same line.

(I assume this is a leftover from refactoring and moving around code.)

Reply:
Not with the format on the end, I can shuffle some stuff around to make it a bit more visually appealing, but I don't think number of lines changes.